src/System.Net.Http/tests/UnitTests/HttpSystemProxyTest.cs

```diff
@@ -64,6 +64,27 @@ public void HttpProxy_SystemProxy_Loaded(string rawProxyString, bool hasInsecure
             }, rawProxyString, hasInsecureProxy.ToString(), hasSecureProxy.ToString()).Dispose();
         }
 
+        [Theory]
+        [InlineData("localhost:1234")]
+        [InlineData("123.123.123.123")]
+        public void HttpProxy_SystemProxy_Loaded(string rawProxyString)
+        {
+            RemoteInvoke((proxyString) =>
+            {
+                IWebProxy p;
+
+                FakeRegistry.Reset();
+
+                FakeRegistry.WinInetProxySettings.Proxy = proxyString;
+                WinInetProxyHelper proxyHelper = new WinInetProxyHelper();
+
+                Assert.True(HttpSystemProxy.TryCreate(out p));
+                Assert.NotNull(p);
```

Comment:
Instead of just Assert'ing for NotNull, you should Assert that the URI created is what you expected.  I.e. from the rawProxyString="localhost:1234", you get "http://localhost:1234" etc.  You would need to add the expected URI string as an additional InlineData parameter.